apps/meteor/app/file-upload/server/lib/FileUpload.ts

```diff
@@ -308,9 +308,20 @@ export const FileUpload = {
 		const store = FileUpload.getStore('Uploads');
 		const image = await store._store.getReadStream(file._id, file);
 
-		const transformer = sharp().resize({ width, height, fit: 'inside' });
+		let transformer = sharp().resize({ width, height, fit: 'inside' });
 
-		const result = transformer.toBuffer({ resolveWithObject: true }).then(({ data, info: { width, height } }) => ({ data, width, height }));
+		if (file.type === 'image/svg+xml') {
+			transformer = transformer.png();
+			if (file.name) {
+				file.name = `${file.name}.png`;
+			}
+		}
+		const result = transformer.toBuffer({ resolveWithObject: true }).then(({ data, info: { width, height, format } }) => ({
+			data,
+			width,
+			height,
+			thumbFile: { ...file, type: mime.lookup(format) || '' } as IUpload,
```

Comment:
Same here, I think we should not return the property thumbFile spreading the file inside of it, and change the type.

Following the previous comment, I think we could change the function return type to something like 

```
{
   result,
   thumbFileName,
   thumbFileType,
}
```